### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by njooma**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,32 +8,21 @@ Build and connect to robots with Flutter
 ## (In)stability Notice
 
 > **Warning**
-> This is an alpha release of the Viam Flutter SDK. The only guarantee is that there **WILL** be breaking changes to the API. We will enumerate these changes in the release notes.
+> This is a beta release of the Viam Flutter SDK. No guarantees are made to the stability of the API. All changes will be enumerated in the release notes.
```
Comment (clintpurser) [inline README.md:11]: 🚨 🚨 🚨 BETA 🚨 🚨 🚨

Comment (jckras) [inline README.md:11]: What is the difference between alpha and beta release?

**Review by clintpurser — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (njooma) [reply README.md:11]: I've saying it's now `beta` since it's published to pub.dev rather than just on github. Otherwise, not functionally different.

Comment (clintpurser) [reply README.md:11]: [see here](https://en.wikipedia.org/wiki/Software_release_life_cycle#:~:text=Alpha%20testing%20is%20the%20first,the%20organization%20that%20developed%20it.)

Alpha is the first phase, typically smaller, lots of iteration. Beta is after, can be a larger group of people testing / slightly more public.

**Review by jckras — APPROVED** [review]
LGTM!  (q) question for my own understanding: Does this mean I need to update the Minimum iOS/android target in the mobile app or is this handled automatically by installing the viam-flutter-sdk? (also, If this is the wrong place to ask that question lmk)

**Comment by njooma** [comment]
@jckras let's talk offline/slack

**Comment by clintpurser** [comment]
> LGTM! (q) question for my own understanding: Does this mean I need to update the Minimum iOS/android target in the mobile app or is this handled automatically by installing the viam-flutter-sdk? (also, If this is the wrong place to ask that question lmk)

@jckras Apps manually track the minimum os versions.

File: README.md (hunk 1 above)
Comment (jckras) [reply README.md:11]: Makes sense, thank you both :)

**Pull request merged**